src/TrackPanelAx.cpp

```diff
@@ -368,33 +363,8 @@ wxAccStatus TrackPanelAx::GetName( int childId, wxString* name )
          }
          else
          {
-            *name = t->GetName();
-            if( *name == t->GetDefaultName() )
-            {
-               /* i18n-hint: The %d is replaced by the number of the track.*/
-               name->Printf(_("Track %d"), TrackNum( t ) );
-            }
-
-            t->TypeSwitch(
-               [&](const LabelTrack *) {
-                  /* i18n-hint: This is for screen reader software and indicates that
-                     this is a Label track.*/
-                  name->Append( wxT(" ") + wxString(_("Label Track")));
-               },
-               [&](const TimeTrack *) {
-                  /* i18n-hint: This is for screen reader software and indicates that
-                     this is a Time track.*/
-                  name->Append( wxT(" ") + wxString(_("Time Track")));
-               }
-#ifdef USE_MIDI
-                ,
-               [&](const NoteTrack *) {
-                  /* i18n-hint: This is for screen reader software and indicates that
-                     this is a Note track.*/
-                  name->Append( wxT(" ") + wxString(_("Note Track")));
```

Comment:
Why was it necessary for resolving this issue, to delete this logic that appends "Label Track", "Note Track", etc. to `name`?

This name is NOT the name you will see on the screen.  It is a name that the screen reader will speak.

@DavidBailes may not agree with this change.

You are removing the Default Name (I didn't like it either) so the suffix "Label Track", etc. so now you are prepending "Track #n" always.  I think that is all right.